ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -391,9 +387,7 @@ void OwInterface::managePanTilt (const string& opname,
 
 bool OwInterface::anglesEquivalent (double deg1, double deg2, double tolerance)
 {
-  return (fabs(deg1 - deg2) <= tolerance ||
-          fabs((deg1 - 360) - deg2) <= tolerance ||
-          fabs((deg1 + 360) - deg2) <= tolerance);
+  return normalize_degrees(fabs(deg1 - deg2)) <= tolerance;
```

Comment:
Possible bug since `normalize_degrees` is capable of being passed a position number and returning a negative number. This becomes more rigorous if we compute the absolute value after normalization. This idiom was my recommendation, so I take full responsibility :laughing:
```suggestion
  return fabs(normalize_degrees(deg1 - deg2)) <= tolerance;
```